Vizzuality/marxan-cloud
# PU grids generation.

### Overview
Job system and first queue + Job to generate Planning units related to a user project area.
Also the testing system for QA this. 

### Testing instructions

Running Tests should pass.
Don't forget to do a clean slate; the bd updates PostGIS to 3.1 and requires refresh the data.
You need to create a Project and automatically a new job is created if the project provides a valid region or extent. That job is going to be processed by the geoprocessing and generate the PU grid. PU grid will only be attached at the scenario level. But the required geometries will be ready on the PU table to be accessible through the project extent.

### Feature relevant tickets

https://vizzuality.atlassian.net/browse/MARXAN-148

---

## Checklist before submitting

- [ ] If this PR adds a feature that should be tested for regressions when
      deploying to staging/production, please add brief testing instructions
      to the deploy checklist (`docs/deployment-checklist.md`)
